android/sdk/src/main/java/com/taobao/weex/utils/WXUtils.java

```diff
@@ -135,7 +135,7 @@ public static Float getFloat(Object value, @Nullable Float df) {
     return df;
   }
 
-  private static float transferWx(String stringWithWXPostfix, int viewport) {
+  public static float transferWx(String stringWithWXPostfix, int viewport) {
```

Comment:
Please mark the following method as `@Deprecated` and `Use {@link WXViewUtils#getRealPxByWidth(Object, float, int)}` instead.
* `getFloat(Object value, @Nullable Float df)`
* `getFloatByViewport(Object value, int viewport) `
* `getFloat(Object value)`